types/index.d.ts

```diff
@@ -0,0 +1,28 @@
+import Vue = require('vue')
+import { WatchOptions } from 'vue'
+import { Observable } from 'rxjs/Observable'
+
+export type Observables = Record<string, Observable<any>>
+declare module 'vue/types/options' {
+  interface ComponentOptions<V extends Vue> {
+    subscriptions?: Observables | ((this: V) => Observables)
+    domStreams?: string[]
```

Comment:
We also have `observableMethods` in component options 😉 

https://github.com/vuejs/vue-rx#createobservablemethodmethodname